# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1370932204 -28800
#      Tue Jun 11 14:30:04 2013 +0800
# Node ID 804d2f7f50bf921346ac0f4c8460a5e6ea8b8043
# Parent  169ea45f260fb0442e750779df1e66c603fd947b
Remove the top horizontal line from the "Workspace" tab.

* libgui/src/settings-dialog.ui: Remove the top horizontal line from the
Workspace tab (line_7).  This is consistent with the other tabs.

diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -1076,29 +1076,16 @@
          <x>10</x>
          <y>8</y>
          <width>631</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
-         <widget class="Line" name="line_7">
-          <property name="minimumSize">
-           <size>
-            <width>0</width>
-            <height>0</height>
-           </size>
-          </property>
-          <property name="orientation">
-           <enum>Qt::Horizontal</enum>
-          </property>
-         </widget>
-        </item>
-        <item>
          <widget class="QGroupBox" name="workspace_colors_box">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
